fe/fe-core/src/main/java/com/starrocks/analysis/Exprs.java

```diff
@@ -0,0 +1,66 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License
+package com.starrocks.analysis;
+
+import com.google.api.client.util.Lists;
+import com.google.common.collect.ImmutableList;
+
+import java.util.List;
+
+public class Exprs {
```

Comment:
it seems that Exprs has nothing different from List<Expr>, why need introduce this class?